bundling/bundle-browser.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -eu
+
+# Cloudflare Workers don't support top-level await
+sed -i~ 's/^if (isDeno)/if (false)/' ../src/platform.deno.ts
+trap 'mv ../src/platform.deno.ts~ ../src/platform.deno.ts' EXIT
+
+echo "Removing old bundle directory"
+rm -rf bundle/out
+
+if [ -d "../out" ]; then
+  echo "Found node files. Skipping d2n step"
+else
+  echo "Running d2n"
+  deno task backport
+fi
+
+echo "Creating bundle folder"
+mkdir -p bundle/out
+
+echo "Generating bundle package.json"
+jq '. + {
+  "name": "@grammyjs/web",
+  "dependencies": {},
+  "scripts": {},
+}' <../package.json >bundle/package.json
+
+echo "Copying node files"
+cp -rv ../LICENSE ../README.md ../out bundle/
+shopt -s globstar
+rm bundle/**/*.js
+
+echo "Bundling source"
+deno bundle ../src/mod.ts bundle/out/mod.js
```

Comment:
… then there is https://deno.land/manual@v1.29.1/tools/bundler#bundling-for-the-web:

_The output of deno bundle is intended for consumption in Deno and not for use in a web browser or other runtimes. That said, depending on the input it may work in other environments._

_If you wish to bundle for the web, we recommend other solutions such as [esbuild](https://esbuild.github.io/)._

Reply:
Esbuild wasn't smart enough to omit `if (false)` when I last tired it. Unlike SWC used by `deno bundle`.

Reply:
`if (false)` is a big hack that should be avoided anyway. Do you remember how we went from bash scripts to d2n? I'd like to make a similar move here, and end up using good tools rather than writing custom scripts.